src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -248,17 +248,23 @@ class LeVADocumentUseCase extends DocGenUseCase {
         def data_ = [
             metadata: this.getDocumentMetadata(this.DOCUMENT_TYPE_NAMES[documentType]),
             data: [
-                // TODO: change template from this.project.repositories to repositories
-                repositories: this.project.repositories,
+                repositories: this.project.repositories.collect { repo_ ->
+                    [
+                        id: repo_.id,
+                        description: repo_.metadata.description,
+                        url: repo_.url
+                    ]
+                },
                 sections: sections,
                 tests: this.project.getAutomatedTestsTypeUnit().collectEntries { testIssue ->
                     [
                         testIssue.key,
                         [
                             key: testIssue.key,
                             description: testIssue.description ?: "",
-                            // TODO: change template from isRelatedTo to systemRequirement
-                            systemRequirement: testIssue.requirements.collect{ it.key }.join(", ")
+                            systemRequirement: testIssue.requirements ? testIssue.requirements.join(", ") : "N/A",
+                            // TODO: should we only link in tech specs where the software design is defined?
```

Comment:
@metmajer  - software design is only applicable for GAMP5 .. here you need to check with QA what they want,...

Reply:
@clemensutschig so then the implementation should be good enough, as we'll issue a `N/A` if none is available

Reply:
Resolved and now considers only those techSpecs that contain a softwareDesignSpec != null